src/accessibilityStatement/AccessibilityStatementFi.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { Fragment } from 'react';
+
+function AccessibilityStatementFi() {
+  return (
+    <Fragment>
+      <h1>Saavutettavuusseloste</h1>
+
+      <p>
+        Tämä saavutettavuusseloste koskee Helsinki profiili- sivustoa. Sivuston
+        osoite on https://hel.fi/profiili
+      </p>
+
+      <h2>Sivustoa koskevat lain säädökset</h2>
+
+      <p>
+        Tämä sivusto on julkaistu 23.9.2018 jälkeen. Sivuston tulee täyttää lain
+        edellyttämät saavutettavuuden vaatimukset.
+      </p>
+
+      <h2>Kaupungin tavoite</h2>
+
+      <p>
+        Digitaalisten palveluiden saavutettavuudessa Helsingin tavoitteena on
+        pyrkiä vähintään WCAG ohjeiston mukaiseen AA- tai sitä parempaan tasoon,
+        mikäli se on kohtuudella mahdollista.
+      </p>
+
+      <h2>Vaatimustenmukaisuustilanne</h2>
+
+      <p>
+        Tämän verkkosivuston saavutettavuuden vaatimustenmukaisuutta ei ole
+        vielä arvioitu.
+      </p>
+
+      <h2>Saavutettavuusselosteen laatiminen</h2>
+
+      <p>Tämä seloste on laadittu 15.4.2020</p>
+
+      <h2>Saavutettavuusselosteen päivittäminen</h2>
+
+      <p>
+        Saavutettavuusseloste päivitetään vastaamaan saavutettavuuden
+        vaatimustenmukaisuudesta tehtyjä havaintoja suoritetun tarkastuksen
+        jälkeen.
+      </p>
+
+      <h2>Tietojen pyytäminen saavutettavassa muodossa</h2>
+      <p>
+        Mikäli käyttäjä ei koe saavansa sivuston sisältöä saavutettavassa
+        muodossa, voi käyttäjä pyytää näitä tietoja sähköpostilla{' '}
+        <a href="mailto:helsinki.palaute@hel.fi">helsinki.palaute@hel.fi</a> tai
+        palautelomakkeella{' '}
+        <a href="https://www.hel.fi/palaute">www.hel.fi/palaute</a>.
+        Tiedusteluun pyritään vastaamaan kohtuullisessa ajassa.
+      </p>
+
+      <h2>Palaute ja yhteystiedot</h2>
+      <p>
+        Kaupunginkanslia, Helsinki
+        <br />
+        +358 9 310 1641
```

Comment:
A `tel:` link is possible here. But maybe it has been omitted for a specific reason?

Reply:
Added `tel:` link.